webui/src/lib/api/index.js

```diff
@@ -671,9 +671,8 @@ export const uploadWithProgress = (url, file, method = 'POST', onProgress = null
                 status: xhr.status,
                 body: xhr.responseText,
                 contentType: xhr.getResponseHeader('Content-Type'),
-                etag: xhr.getResponseHeader('ETag'),
-                contentMD5: xhr.getResponseHeader('Content-MD5'),
```

Comment:
Can you tell why this isn't working?

Reply:
From what I understood, the put request for the object to the presigned url, returns a response with raw headers, these ones are harder to extract and need to be parsed manually.